<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameViewer</class>
 <widget class="QWidget" name="FrameViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_x">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_y1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="ImageViewNoKeyboard" name="imageView" native="true"/>
      <widget class="ProjectionWidget" name="wiProfileX" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_y2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="ProjectionWidget" name="wiProfileY" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewNoKeyboard</class>
   <extends>QWidget</extends>
   <header>widgets.image_view</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectionWidget</class>
   <extends>QWidget</extends>
   <header>widgets.projectionwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
